src/scripts/main.js

```diff
@@ -1,3 +1,66 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('tbody');
+let allRows = document.querySelectorAll('tr');
```

Comment:
```suggestion
let allRows = tbody.querySelectorAll('tr');
```
it' better to use `tbody` instead of `document` in this task